src/content/docs/tutorial-errors/reduce-common-errors.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Reduce common errors
+metaDescription: Learn how to manage a high number of errors in your app.   
+---
+
+So far, you've learned how to scope out a service outage and get the error in front of the correct teams. Error analysis doesn't end after the solution is deployed, however. To return to our flooded yard metaphor, you've only diagnosed the leaking pipe as the direct cause of the flood, then ordered a patch to fix it. You aren't done, though. At this point, you should ask yourself: What caused the leaking pipe to begin with? 
+
+## Objectives [#objectives]
+
+This tutorial helps you understand the difference between direct cause and root cause so you can reduce common errors over time. By the end of this tutorial, you will be able to:
+
+* Route errors to the correct teams
+* Diagnose the root cause of an incident
+
+## Route errors to the correct teams [#route]
+
+To send errors to other teams, you need to return to the **Errors** summary page. If you don't own a component or functionality, you can use this page to send grouped errors to the correct teams.  
+
+    <img
+        title="Overview errors affecting your services"
+        alt="A screenshot showing an app with many errors"
+        src={apmTriageError}
+    />  
+
+In the previous tutorial docs, we covered how to quickly respond to a critical error. But even without a service outage, we recommend that you return to the **Errors** page for at least 10 minutes on a weekly cadence. 
+
+* Imagine a patch that was deployed 48 hours ago on a service. During your weekly triage, you notice a new error has appeared in your errors inbox. An alert hasn't fired because the error occurence hasn't met your alert threshold, but you do notice an increase in error incidences. 
```

Comment:
```suggestion
* Imagine a patch that was deployed 48 hours ago on a service. During your weekly triage, you notice a new error has appeared in your errors inbox. An alert hasn't fired because the number of error occurrences hasn't met your alert threshold, but you do notice an increase in error occurrences. 
```